cookbook/specs/kakarot.json

```diff
@@ -0,0 +1,73 @@
+{
+    "proposal": {
+      "title": "Add Specs: Kakarot",
+      "description": "Adding new specification support for relaying data on Kakarot",
+      "specs": [
+        {
+          "index": "KAKAROTT",
+          "name": "kakarot sepolia testnet",
+          "enabled": true,
+          "imports": ["ETH1"],
+          "reliability_threshold": 268435455,
+          "data_reliability_enabled": true,
+          "block_distance_for_finalized_data": 1,
+          "blocks_in_finalization_proof": 3,
+          "average_block_time": 10200,
+          "allowed_block_lag_for_qos_sync": 5,
+          "shares": 1,
+          "min_stake_provider": {
+            "denom": "ulava",
+            "amount": "47500000000"
+          },
+          "api_collections": [
+            {
+              "enabled": true,
+              "collection_data": {
+                "api_interface": "jsonrpc",
+                "internal_path": "",
+                "type": "POST",
+                "add_on": ""
+              },
+              "apis": [],
+              "headers": [],
+              "inheritance_apis": [],
+              "parse_directives": [],
+              "verifications": [
+                {
+                  "name": "chain-id",
+                  "values": [
+                    {
+                      "expected_value": "0x6b6b7274"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "latest_distance": 847
```

Comment:
pruning should be for 24H of blocks
24*60*60/average_block_time = 8640

Reply:
Resolved in fix

Reply:
https://github.com/lavanet/lava/pull/1447/commits/86e3706b61d94ad14dd755e8fd2445de9808564b

see here